tests/tools/test_write_code.py

```
@@ -0,0 +1,383 @@
+import pytest
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)